pkg/reconciler/delivery/table_test.go

```diff
@@ -44,73 +49,83 @@ func TestReconcile(t *testing.T) {
 		Key: "too/many/parts",
 	}, {
 		Name: "does nothing when event refers to KCD",
-		Key:  "default/test",
+		Key:  "default/test1",
 		Objects: []runtime.Object{
-			Route("default", "test", WithConfigTarget("test"), WithRouteGeneration(1)),
+			Route("default", "test1", WithConfigTarget("test1"), WithRouteGeneration(1)),
 			Configuration(KCDNamespace, KCDName),
 		},
 	}, {
 		Name: "does nothing when latest created is not ready",
-		Key:  "default/test",
+		Key:  "default/test2",
 		Objects: []runtime.Object{
-			Route("default", "test", WithConfigTarget("test"), WithRouteGeneration(1)),
-			Configuration("default", "test", WithLatestCreated("rev-1")),
+			Route("default", "test2", WithConfigTarget("test2"), WithRouteGeneration(1)),
+			Configuration("default", "test2", WithLatestCreated("rev-1")),
 		},
 	}, {
 		Name: "sets the route to 100% if the configuration is ready",
```

Comment:
Should this assert there are no events queued? I think it makes sense to assert the queue response on every test.